ui/narrative/methods/import_genbank_as_genome_from_staging/spec.json

```diff
@@ -128,12 +128,8 @@
           "target_property": "genetic_code"
         },
         {
-          "input_parameter": "generate_ids_if_needed",
-          "target_property": "generate_ids_if_needed"
-        },
-        {
-          "input_parameter": "exclude_ontologies",
-          "target_property": "exclude_ontologies"
+          "input_parameter": "generate_missing_genes",
```

Comment:
GFU will take 'generate_missing_genes' and ignore 'exclude_ontologies'? Was is just a parameter name change for GFU? I don't see GFU is taking generate_missing_genes, at least not for master branch.